run-tests.sh

```diff
@@ -9,7 +9,7 @@
 docker-services-cli up postgresql es redis && \
```

Comment:
```suggestion
docker-services-cli --verbose up es postgresql redis && \
```
using --verbose mode